playground/mocks/data/idp/idx/enroll-profile-new-custom-labels.json

```diff
@@ -21,17 +21,20 @@
                 {
                   "name": "lastName",
                   "label": "Please enter your last name",
-                  "required": true
+                  "required": true,
+                  "customLabel": true
```

Comment:
No, I created a ticket for it to be handled in Gen3, but this change is reflected here because I merged from master into this branch. Its just reflecting that this change didnt exist in the 0.5 branch.